static/js/user_group_pill.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+function display_pill(group) {
+    return group.name + ": " + group.members.size + " users";
+}
+
+exports.create_item_from_group_name = function (group_name, current_items) {
+    group_name = group_name.trim();
+    const group = user_groups.get_user_group_from_name(group_name);
+    if (!group) {
+        return undefined;
+    }
+
+    const existing_ids = current_items.map((item) => item.group_id);
+    if (existing_ids.includes(group.id)) {
+        return undefined;
+    }
```

Comment:
can this be rewritten as below? I suppose it may yield tiny bit improvement as find returns as soon as first match is found
````Javascript
const is_group_in_current_items = current_items.find(item => item.group_id === group.id)
if (is_group_in_current_iems) {
  return undefined;
}
````